_config.yml

```diff
@@ -13,7 +13,7 @@ navbar-links:
   Map: "/map"
   Radio Files: "/files"
   Logging: "/logging"
-  Chat: "https://discord.com/invite/mqkD4fTjrn"
+  Chat: "https://discord.gg/3ZgBkv59ZC"
```

Comment:
The link currently on the website seems to work for me. Does it not work for y'all ?